### PR TITLE
OV-164: Remove sx prop

### PR DESCRIPTION
**Pull request by stefano-lacorazza**
Removed all sx props and added the styles to module.css files.

### CODE REVIEW

**Comment by stefano-lacorazza** [comment]
Is it ok to name the module. css files as: componentName.module.css or is it better to just call them styles.module.css as each file is in its component folder?

**Comment by anton-otroshchenko** [comment]
We should create styles file for each component

File: frontend/src/bundles/common/components/video-modal/components/video-modal-content/components/generate-script-form/generate-script-form.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import {
     generateVideoScriptValidationSchema,
 } from '~/bundles/video-scripts/video-scripts.js';
 
+import styles from '../../../../video-modal.module.css';
```
Comment (o-nedashkivska) [inline frontend/src/bundles/common/components/video-modal/components/video-modal-content/components/generate-script-form/generate-script-form.tsx:15]: Let's import from `~/bundles ...` here

Comment (o-nedashkivska) [reply frontend/src/bundles/common/components/video-modal/components/video-modal-content/components/generate-script-form/generate-script-form.tsx:15]: And the same for components below

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
all css module file should have same name `styles.module.css`

**Review by o-nedashkivska — DISMISSED** [review]

File: frontend/src/bundles/studio/components/video-menu/components/menu/menu.module.css
Comment (XCODE89) [inline frontend/src/bundles/studio/components/video-menu/components/menu/menu.module.css:1]: menu.module.css to styles.module.css

**Pull request merged**